feature_tests/features/producer/producer-createDocumentPointer-success.feature

```diff
@@ -160,3 +161,73 @@ Feature: Producer Create Success scenarios
       | updated_on  | NULL                                    |
       | document    | <document>                              |
       | created_on  | <timestamp>                             |
+
+  Scenario: Successfully create a Document Pointer for one directional data sync when date is provided
```

Comment:
I am not able to see any differences between the below scenarios apart from the scenario titles? Am i missing something here?     

- Successfully create a Document Pointer for one directional data sync when no date is provided
- Successfully create a Document Pointer for one directional data sync when date is provided